doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1808,6 +1808,7 @@ Plotting
 Groupby/Resample/Rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Bug in :func:`pandas.core.window.Rolling.min` and :func:`pandas.core.window.Rolling.max` with ``closed='left'``, a datetime-like index and only one entry in the series leading to segfault (:issue:`24718`)
```

Comment:
Is this truly this specific? Off the top of my head seems strange that this would only be applicable to date time like indices